examples/tess/abc_setup.jl

```diff
@@ -0,0 +1,271 @@
+## ExoplanetsSysSim/examples/tess/abc_setup.jl
```

Comment:
I haven't reviewed this closely, but it looks like it's largely a copy and paste, but then replacing with tess config settings.  Let me know if you have any questions you'd like me to take a look at.